Client/Users/index.ts

```diff
@@ -25,7 +25,7 @@ export class Users extends Collection<model.UserResponse, model.UserSearchReques
 		)
 		return this.extractResponse(response)
 	}
-	async getCategory(): Promise<string[] | model.ErrorResponse> {
+	async getUserCategory(): Promise<string[] | model.ErrorResponse> {
 		const result = await this.connection.get<string[]>(`users/category`)
 		return result
 	}
```

Comment:
please don't use this method - use `GET /category` to do the same thing (get the categories)

Reply:
we have this in category folder so just gonna delete this one from users (already checked no place is using this)